server/sonar-web/src/main/js/components/icons-components/ChevronDownIcon.tsx

```diff
@@ -17,12 +17,27 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-const Handlebars = require('handlebars/runtime');
-const { translate } = require('../../helpers/l10n');
+import * as React from 'react';
+import { IconProps } from './types';
 
-module.exports = function(severity) {
-  return new Handlebars.default.SafeString(
-    `<i class="icon-severity-${severity.toLowerCase()}"></i>` +
-      `&nbsp;${translate('severity', severity)}`
+export default function ChevronDownIcon({
+  className,
+  fill = 'currentColor',
+  size = 16
+}: IconProps) {
+  return (
+    <svg
+      className={className}
+      height={size}
+      version="1.1"
+      viewBox="0 0 16 16"
+      width={size}
+      xmlSpace="preserve"
+      xmlnsXlink="http://www.w3.org/1999/xlink">
+      <path
+        d="M7.72 11.596L3.119 6.992A.382.382 0 0 1 3 6.713c0-.108.04-.2.118-.279l1.03-1.03a.382.382 0 0 1 .278-.117c.108 0 .201.04.28.117L8 8.7l3.294-3.295a.382.382 0 0 1 .28-.117c.108 0 .2.04.279.117l1.03 1.03a.382.382 0 0 1 .117.28c0 .107-.04.2-.118.278L8.28 11.596a.382.382 0 0 1-.279.117.382.382 0 0 1-.28-.117z"
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Exceeds maximum line length of 180 ([typescript:S103](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS103))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&pullRequest=3154&issues=AWIvgLgl_wH-bYHl3a4m&open=AWIvgLgl_wH-bYHl3a4m)

[ISSUE_KEY]: AWIvgLgl_wH-bYHl3a4m